src/scheduler.c

```diff
@@ -683,6 +684,10 @@ zend_bool php_parallel_scheduler_cancel(php_parallel_future_t *future) {
     }
 }
 
+PARALLEL_API zend_bool php_parallel_is_parallel_worker_thread(void) {
```

Comment:
Consistency: you used `bool` for `php_parallel_scheduler_empty` but used `zend_bool` here for `php_parallel_is_parallel_worker_thread`.

Reply:
```suggestion
PARALLEL_API bool php_parallel_is_parallel_worker_thread(void) {
```
Nice catch, that should be `bool` as well, see https://github.com/krakjoe/parallel/pull/353